Makefile

```diff
@@ -116,6 +116,19 @@ cert-mgr: ## Install the certification manager
 	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/$(CERT_MGR_VERSION)/cert-manager.yaml
 	kubectl wait --for=condition=Available --namespace=cert-manager deployment/cert-manager-webhook --timeout=60s
 
+uninstall: ## Uninstall all rukpak-related content from the cluster
+	kubectl delete ns rukpak-system
+	kubectl delete ns crdvalidator-system
+	kubectl delete ns cert-manager
+	kubectl delete crd bundles.core.rukpak.io
+	kubectl delete crd bundleinstances.core.rukpak.io
+	kubectl delete crd certificaterequests.cert-manager.io
+	kubectl delete crd certificates.cert-manager.io
+	kubectl delete crd challenges.acme.cert-manager.io
+	kubectl delete crd clusterissuers.cert-manager.io
+	kubectl delete crd issuers.cert-manager.io
+	kubectl delete crd orders.acme.cert-manager.io
```

Comment:
The following ValidatingWebhookConfigurations remain after this.
```
cert-manager-webhook 
crd-validation-webhook 
rukpak-webhook
```
They need to be cleaned, too.  The `crd-validation-webhook` causes an install error when the cert-manager is reinstalled.

Reply:
Sounds good, will update